node-tests/__snapshots__/util.test.js.snap

```diff
@@ -0,0 +1,61 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`util @generatePreviewHead should work with file created with \`ember build\` 1`] = `
```

Comment:
Maybe we should not commit this file WDYT? LGTM otherwise

Reply:
We need these snapshot files as they are required to ensure no one changes the expectations of the test output unless we approve it. https://jestjs.io/docs/snapshot-testing